File: src/lugbygame/Unit.java
package lugbygame;

public class Unit {
	protected int x;
	protected int y;

	public Unit(int x, int y) {
		this.x = x;
		this.y = y;
	}

	public Unit(Unit u) {
		this.x = u.x;
		this.y = u.y;
	}

	public Unit(Unit u, boolean mirror) {
		this.x = _Game.WIDTH - u.x - 1;
		this.y = u.y;
	}

	public int getDistance(Unit u) {
		return (int) Math.sqrt(Math.pow((x - u.x), 2) + Math.pow(y - u.y, 2));
	}
}

File: lucby_edit/Unit.py
import math

class Unit:
	x=y=0
	def __init__(self,u=None,x: int =None,y: int =None, mirror=None):
		if u!=None:
			if mirror!=None:
				self.x = int(64- u.x) - 1
				self.y = int(u.y)
			else:
				self.x=int(u.x)
				self.y=int(u.y)
		else:
			self.x=int(x)
			self.y=int(y)
	def getDistance(self, u):
		return int(math.sqrt(math.pow(self.x-u.x, 2)+math.pow(self.y-u.y, 2)))
	def getX(self):
		return int(self.x)
	def getY(self):
		return int(self.y)
	